oslabs-beta/DockerLocal
# Security

## Description:
- These changes enhance the security of the app. I followed [this security checklist](https://www.electronjs.org/docs/tutorial/security) from the Electron documentation. Also fixed TypeScript type errors.
# Changes I Made:
- Index.ts updated to follow electron security guidelines
- Update server.ts  CORS policy to only allow CORS requests from our front-end server
- Disables server.ts 'x-powered-by' header
- Content-Security-Policy added to Index.html- only allows sources we're using and inline styles
- Fix typescript errors in fetchReopHelpers and ProjectPage.tsx 
- Moved types to types.ts file
- Export the Express server URL (localhost:3001) to a constant
- Remove unused google font import on App.sass

## How to Test:
- npm install and npm start to launch app
- Login and all existing functionality should work- saving and loading project list, adding and cloning repos, compose yaml file
- Electron security warning no longer shows in the console on startup
